apis/v1/controllers/addresses_controller.js

```diff
@@ -203,8 +203,6 @@ exports.createAddress = async (req, res, next) => {
       throw new Error("No User matches this id");
     }
 
-
-
     getKeyFromValue(statusMap, addressData.status)
```

Comment:
This returns a value. if it is not being used. Then it should be removed.

Reply:
this was approved instead of requesting changes. Please ensure these are addressed in the next review